### DISCUSSION TITLE
Navigation Timeout

### DISCUSSION THREAD

**Discussion by bugrakdogan** [jef/streetmerchant#2253] [Help]
Hey guys,

I've been looking for a solution for 2 days now. When I disable headless pages are I see that pages are opening without any issues. But I get timeout erros in SM.  I have tried everything I can think of (e.g. changing dns, changing ip, reseting computer, changing almost every setting in dotenv) but nothing happened. I've been looking for help in discord for 2 days and even though ShawTheatre was very helpfull we could not be able to get it to work. Can you please help me? 

@jef I'd be glad if you help

![image](https://user-images.githubusercontent.com/32468500/113049208-09380200-91ac-11eb-9abe-fb1340c58b29.png)

**Comment by xmpierce1** [comment]
I have the same issues. I know its something to do with the proxies because when I do not use the proxies I do not have an issue.
![image](https://user-images.githubusercontent.com/74942520/113730266-bb4c5e00-96c5-11eb-8e6a-c702ee5e60d7.png)

**Comment by ryanschieber** [comment]
Hi, 

I'd suggest to increase `PAGE_SLEEP_MIN=`. For me, a value of 120k works fine.
